index.js

```diff
@@ -121,7 +121,7 @@ async function installCrystalForLinux({crystal, shards, arch = getArch(), path})
     checkArch(arch, Object.keys(filePatterns));
 
     const depsTask = installAptPackages(
-        "libevent-dev libgmp-dev libpcre3-dev libssl-dev libxml2-dev libyaml-dev".split(" "),
+        "libevent-dev libgmp-dev libpcre2-dev libssl-dev libxml2-dev libyaml-dev".split(" "),
```

Comment:
Actually, we need to install `libpcre` as well. At least for releases < 1.8. But it's probably okay to install both for a while.

```suggestion
        "libevent-dev libgmp-dev libpcre2-dev libpcre3-dev libssl-dev libxml2-dev libyaml-dev".split(" "),
```

Reply:
More elegant solution would be to skip `libpcre2` for versions < 1.8.

Reply:
ah OK, I was already doing the first, but my lack of knowledge in JS is already making me nervous, so 👍 to installing both 😅